vunit/nvc_interface.py

```diff
@@ -51,7 +51,8 @@ def setup_library_mapping(self, project, vhdl_standard):
         """
         Setup the library mapping according to project
         """
-
+        project.add_external_library("ieee_proposed",
```

Comment:
It is not really recommended to add libraries here. This would be something nvc users would put in their run.py if they require it.